test/index.spec.js

```diff
@@ -10,7 +10,11 @@ describe('discipl-law-reg', () => {
   describe('The discipl-law-reg library', () => {
     it('test pegjs generator', async () => {
       let parsedFact = lawReg.evaluateFactFunction('(NIET [fact1]) OF ([fact2] EN [fact3])')
+      let parsedFact2 = lawReg.evaluateFactFunction('[fact1] EN [fact2] EN [fact3]')
```

Comment:
So, personally, I would split this into multiple tests, and describe in each test what you're testing. 
So that means contents of it like `it('correctly parses a single fact'`, `it('correctly parses a multiple-AND construction'`, `it('correctly parses a complex epression'`.

The function of this, is if someone changes the grammar and breaks a test, they will know what usecase they broke.

I would also remove the console.log statements below. It clutters the test output and the expect() exactly describes the contents, so you don't even need to run  the test to see it.